From 1a543bdcc4c34931aaa111f3d20e2220e715bfe7 Mon Sep 17 00:00:00 2001
From: Pinkesh Vaghela <pinkesh.vaghela@einfochips.com>
Date: Fri, 29 Nov 2024 10:14:57 +0000
Subject: [PATCH 117/128] UBUNTU: Add dependency for eswin-eic7x-gpu

Add dependency of eswin-eic7x-gpu package version to ensure updated
firmware for updated imagination GPU driver

Signed-off-by: Pinkesh Vaghela <pinkesh.vaghela@einfochips.com>
---
 debian/control.d/flavour-control.stub | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control.d/flavour-control.stub b/debian/control.d/flavour-control.stub
index d895fa59bda1..cc30c87de207 100644
--- a/debian/control.d/flavour-control.stub
+++ b/debian/control.d/flavour-control.stub
@@ -45,7 +45,7 @@ Build-Profiles: <!stage1>
 Architecture: ARCH
 Section: kernel
 Priority: optional
-Depends: ${misc:Depends}, ${shlibs:Depends}
+Depends: ${misc:Depends}, ${shlibs:Depends}, eswin-eic7x-gpu (>= 24.2+0ubuntu11)
 Built-Using: ${linux:BuiltUsing}
 Description: Linux kernel modules for version PKGVER on DESC
  Contains the corresponding System.map file, the modules built by the
-- 
2.47.0

